18F/identity-idp
# LG-6193: Integrate IdV app personal key step into proofing flow (feature-flagged)

Extracted from: #6229

This is basically the same as #6229, with the exception that the feature flag is not yet enabled. This means that you can reach the step only when changing the feature flag yourself, then navigating through the proofing flow.

The reason to separate this from #6229 is that the changes in #6229 became inflated to include both (a) the integration of the step and (b) numerous fixes to support proofing with JavaScript enabled for the feature specs.

**Testing Instructions:**

1. **Set `idv_api_enabled: true` in your local `config/application.yml`**
2. Navigate to http://localhost:3000
3. Sign in
4. Navigate to http://localhost:3000/verify
5. Complete proofing process up to and including the password entry step
6. After "Continue"-ing from password entry step, observe...
   - URL is "/verify/v2"
   - Real personal key is shown on page
   - You can enter personal key and continue to post-proofing flow step (account page if session associated with SP, otherwise confirmation screen)

**Next Steps:**

Merging this separate from #6229 would allow rebasing to change the focus of #6229 to improving spec compatibility for JavaScript-enabled proofing.
